color_helper.py

```diff
@@ -1001,15 +1001,32 @@ def is_enabled(self, mode, palette_name=None, color=None, auto=False):
         )
 
 
-class ColorHelperFileIndexCommand(sublime_plugin.TextCommand):
-    """Color Helper file index command."""
+class ColorHelperFileIndexCommand(sublime_plugin.WindowCommand):
+    """Forces Color Helper to index the colors in the current file.
 
-    def run(self, edit):
+    If `scan_scopes` argument is provided, instead of using the normal
+    configuration it will look for colors inside the given scopes.
+    """
+
+    def run(self, scan_scopes=None, allowed_colors=("all"), use_hex_argb=False):
```

Comment:
Unfortunately, `("all")` is just a string in parenthesis, to make a single length tuple, you need to have a trailing comma: `("all",)`.

Reply:
sorry typo